metricbeat/tests/system/metricbeat.py

```diff
@@ -52,3 +52,15 @@ def de_dot(self, existing_fields):
                 fields[key] = self.de_dot(fields[key])
 
         return fields
+
+    def assert_no_logged_warnings(self):
+        """
+        Assert that the log file contains no ERR or WARN lines.
+        """
+        log = self.get_log()
+        log = log.replace("WARN EXPERIMENTAL", "")
```

Comment:
Let me fix it up to get it back to the three separate commits. Then you can use "rebase and merge".